package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "build": "webpack --mode production",
     "lint": "eslint .",
-    "start": "node .",
+    "start": "npm run build && node server/index.js",
```

Comment:
This will cause now to fail to deploy.  `npm start` can not include a build.

Reply:
We changed that so that we could be able to work on our localhost.  It wouldn't run using 'node .'